trxer.go

```diff
@@ -18,7 +29,161 @@ type measurement struct {
 	time  float64
 }
 
-func udp_client_worker(addr string, wg sync.WaitGroup) {
+func quic_client_worker(addr string, wg *sync.WaitGroup) {
+	fmt.Println("Quic stream connecting to: ", addr)
+
+	buf := make([]byte, 1400, 1400)
```

Comment:
Probably this is a performance problem! You loop and sent 1400 byte in one call. The UDP code uses `BYTE_BUFFER_SIZE` which is quite larger.

This code should not just send 1400 bytes, just use BYTE_BUFFER_SIZE.

Better: we define a new application option (e.g. --call-size <n>) and let the user define the actual size in bytes.